infra/environments/main.tf

```diff
@@ -25,6 +30,16 @@ terraform {
 }
 
 
+module "mantarray_downloads" {
```

Comment:
this isn't specific to 'mantarray', all Curi

Reply:
renamed